src/components/IntegrationTests/IntegrationTestsListView/IntegrationTestsListView.tsx

```diff
@@ -1,150 +1,133 @@
 import { useAccessReviewForModel } from '../../../utils/rbac';
 import { ButtonWithAccessTooltip } from '../../ButtonWithAccessTooltip';
 import { IntegrationTestListHeader } from './IntegrationTestListHeader';
 import IntegrationTestListRow from './IntegrationTestListRow';
+
 const IntegrationTestsEmptyState: React.FC<
   React.PropsWithChildren<{
-    handleAddTest: () => void;
-    canCreateIntegrationTest: boolean;
+    applicationName: string;
   }>
-> = ({ handleAddTest, canCreateIntegrationTest }) => {
-  return (
-    <AppEmptyState
-      data-test="integration-tests__empty"
-      emptyStateImg={emptyStateImgUrl}
-      title="Test any code changes"
-    >
-      <EmptyStateBody>
-        Integration tests run in parallel, validating each new component build with the latest
-        version of all other application components.
-        <br />
-        To add an integration test, link to a Git repository containing code that can test how your
-        application components work together.
-      </EmptyStateBody>
-      <EmptyStateActions>
-        <ButtonWithAccessTooltip
-          variant={ButtonVariant.primary}
-          onClick={handleAddTest}
-          isDisabled={!canCreateIntegrationTest}
-          tooltip="You don't have access to add an integration test"
-          data-test="add-integration-test"
-        >
-          Add integration test
-        </ButtonWithAccessTooltip>
-      </EmptyStateActions>
-    </AppEmptyState>
-  );
-};
+> = ({ applicationName }) => (
+  <EmptyState>
+    <EmptyStateBody>
+      No integration tests found for application <strong>{applicationName}</strong>.
+    </EmptyStateBody>
+    <EmptyStateActions>
+      <Button variant="primary" onClick={() => window.location.reload()}>
+        Reload
+      </Button>
+    </EmptyStateActions>
+  </EmptyState>
+);
 
-const IntegrationTestsListView: React.FC<React.PropsWithChildren> = () => {
-  const { applicationName } = useParams<RouterParams>();
+export const IntegrationTestsListView: React.FC = () => {
   const namespace = useNamespace();
-  const [canCreateIntegrationTest] = useAccessReviewForModel(
-    IntegrationTestScenarioModel,
-    'create',
-  );
-
+  const { applicationName } = useParams<RouterParams>();
   const navigate = useNavigate();
-  const [integrationTests, integrationTestsLoaded] = useIntegrationTestScenarios(
+  const [integrationTests, integrationTestsLoaded, error] = useIntegrationTestScenarios(
     namespace,
     applicationName,
   );
   const { filters: unparsedFilters, setFilters, onClearFilters } = React.useContext(FilterContext);
   const filters = useDeepCompareMemoize({
     name: unparsedFilters.name ? (unparsedFilters.name as string) : '',
   });
-  const { name: nameFilter } = filters;
+
+  const { name } = filters;
 
   const filteredIntegrationTests = React.useMemo(
     () =>
-      nameFilter
-        ? integrationTests.filter((test) => test.metadata.name.indexOf(nameFilter) !== -1)
```

Comment:
if you want to make the changes to allow case insensitive in search filter, you just need to change this line that's all.